_posts/pl/2021-02-01-wszystko-co-chcielibyscie-wiedziec-o-fontach-ale-baliscie-sie-zapytac.md

```diff
@@ -0,0 +1,104 @@
+---
+layout:    post
+title:     Wszystko, co chcielibyście wiedzieć o fontach, ale baliście się zapytać
+published: false
+date:      2021-01-01 09:00:00 +0100
+author:    pgrobelny
+lang:      pl
+image:     
+tags:
+    - font
+    - czcionka
+    - rem
+    - woff
+    - font-face
+---
+
+Większość osób, które pracowały na frontendzie na pewno miały do czynienia z fontami, czy to wrzucały je do projektu,
+czy po prostu zmieniały sposób, w jaki tekst wyświetla się na stronie. Na pierwszy rzut oka może się wydawać, że to bajecznie proste.
+Jest jednak kilka ważnych rzeczy, na które warto zwracać uwagę i być ich świadomym.
+
+## Sposoby określania wielkości fontów (EM, REM, PX)
+
+Zacznijmy od początku. Jak poprawnie zdefiniować wielkość fontów? W większości przypadków wyboru dokonujemy pomiędzy PX, EM a REM.
```

Comment:
Zacznijmy od początku brzmi trochę jak pleonazm, bo raczej trudno zacząć od środka, natomiast to raczej luźna uwaga

Reply:
można też zacząć od końca :)